lldb/test/API/python_api/watchpoint/watchlocation/TestTargetWatchAddress.py

```diff
@@ -135,5 +135,5 @@ def test_watch_address_with_invalid_watch_size(self):
             self.expect(
                 error.GetCString(),
                 exe=False,
-                substrs=["watch size of %d is not supported" % 365],
+                substrs=["Setting one of the watchpoint resources failed"],
```

Comment:
Is the original error still relevant? Should it be one of the watchpoint resources failed because...

Reply:
This test is trying to watch a 365 byte region, it was intended to detect the fact that you can only watch 1/2/4/8 byte blocks of memory (the --size was really an enum and it had to be one of those).  With this patch currently, lldb will accept the 365 byte watch request and pass it to the remote stub, which will probably return an error.  (debugserver will actually break it into aligned watchable regions and do the right thing). When I finish the large watchpoint work, lldb will break this 365 byte request into 46 8-byte regions (for linux with BAS only watchpoints) and when we try to set the 5th one (or however many the core actually supports), that watch request will fail.  Maybe the error message should say something about how your watchpoint was broken into <x> watchpoint requests but only <y> of them could be set.

Reply:
Yeah I don't think it would need to be super specific, that is what logs are for. Drawing the distinction between failed because the stub didn't have resources, and failed because the stub exploded, that's more what I'd like to see.

And here the original error message is actually still pretty vague. It tells me that size N isn't support, but not what is supported (which is a whole other thing).